backend/backend.go

```diff
@@ -291,6 +294,7 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 	backend.socksProxy = backendProxy
 	backend.httpClient = hclient
 	backend.etherScanHTTPClient = hclient
+	backend.blockbookHTTPClient = hclient
```

Comment:
Maybe in the past these were different, but now that they are all the same, it would make sense to just use `backend.httpClient` and remove the other two, right?

Reply:
Yep, done!